numpy/distutils/fcompiler/__init__.py

```diff
@@ -744,7 +744,7 @@ def wrap_unlinkable_objects(self, objects, output_dir, extra_dll_dir):
                'intelvem', 'intelem', 'flang')),
     ('cygwin.*', ('gnu', 'intelv', 'absoft', 'compaqv', 'intelev', 'gnu95', 'g95')),
     ('linux.*', ('gnu95', 'intel', 'lahey', 'pg', 'nv', 'absoft', 'nag', 'vast', 'compaq',
-                 'intele', 'intelem', 'gnu', 'g95', 'pathf95', 'nagfor')),
+                 'intele', 'intelem', 'gnu', 'g95', 'pathf95', 'nagfor', 'fj')),
```

Comment:
Can you call it either `frt` (after the executable) or `fujitsu`? Just `fj` is not descriptive enough.

Reply:
Thank you for confirmation. From now on, the name of the compiler type will be fixed at fujitsu.